rust/agama-dbus-server/src/main.rs

```diff
@@ -9,6 +10,7 @@ const ADDRESS: &str = "unix:path=/run/agama/bus";
 #[async_std::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     // When adding more services here, the order might be important.
+    crate::questions::start_service(ADDRESS).await?;
```

Comment:
Three lines and all of them are different :)

* The first line uses the full path to call a function (`start_service`).
* The second line calls an associated function (`start`) of a struct (`LocaleService`) using its full path. However, it uses the `let _con` trick (because it does not own the connection), which looks obscure to me.
* The third one is a combination: it calls an associated function (`start`) of a struct (`NetworkService`) that is previously imported (no full path).

Let's think about having a style guide. Otherwise, the code will get out of control pretty quickly.

Reply:
agreed. To be honest here when I need to start service I do not justify to me why it needs anything to own so that is why I do not use struct...at least I will try to unify locale and and here questions to looks similar.